### DISCUSSION TITLE
Remix build error without log info

### DISCUSSION THREAD

**Discussion by netanelben** [vercel/community#5215] [Help]
### Summary

Hello 👋🏽 
i'm new to Remix and Vercel and need some help!

 *@vercel/remix": "^2.4.0"* 

Using the template provided to deploy the app, while receiving this error and can't get further information..
would appreciate some help investigating this err since I couldn't get any log info.

### Steps to Reproduce

Locally builds 🆗 
I can provide more details about the app if needed.

Received Error:

> Build Failed
> An unexpected error happened when running this build. We have been notified of the problem. If you have any questions, please > contact Vercel Support https://vercel.com/help.

- deploy hash: `AnznFGeQsstJK8QtuaV6y6QeGpGW`

**Comment by amyegan** [comment]
Hey @netanelben. Which template did you use? I want to be sure I investigate the right one. 🙏

**Reply by netanelben** [reply]
that is the one https://github.com/vercel/vercel/tree/main/examples/remix :)

**Reply by netanelben** [reply]
@amyegan 🆙 
_mention in case you missed my reply :)_

**Reply by amyegan** [reply]
Thanks for sharing that @netanelben! 🙂 I'm a little behind so I appreciate you waiting. I just deployed from that template and the build didn't throw any errors for me. Is there code change or package I should add to match your project?

When a local build succeeds but server build fails, I usually try [deploying from a local build (i.e. prebuilt deployment)](https://vercel.com/docs/cli/deploying-from-cli#deploying-from-local-build-prebuilt). If that succeeds, then the problem might be a missing dependency.

**Reply by netanelben** [reply]
@amyegan I will try the local build deploy command, but, there is a way to output the machine log somehow?
instead of this message: *Build Failed, An unexpected error happened when running this build...* in order to investigate the issue.

**Reply by amyegan** [reply]
If you run [`vercel build`](https://vercel.com/docs/cli/build#) on local and it fails, you can add the [`--debug` option](https://vercel.com/docs/cli/global-options#debug) to log additional build info
